wgsl/index.bs

```diff
@@ -9338,6 +9338,14 @@ struct __modf_result_vecN {
     [=Component-wise=] when |T| is a vector.
     (GLSLstd450SClamp)
 
+  <tr algorithm="count leading zeroes">
+    <td>|T| is [INTEGRAL]
+    <td class="nowrap">`countLeadingZeros(`|e|`:` |T| `) ->` |T|
+    <td>The number of consectuive 0 bits starting from the most significant bit positions
+        of |e|, when |T| is a scalar type.<br>
```

Comment:
```suggestion
    <td>The number of consecutive 0 bits starting from the most significant bit
        of |e|, when |T| is a scalar type.<br>
```

I think it reads better by dropping "positions". Also minor typo in consecutive.

Reply:
done